src/components/node-list/styles/_row.scss

```diff
@@ -114,39 +95,16 @@
   white-space: nowrap;
   text-overflow: ellipsis;
 
-  .kui-theme--light & {
-    color: rgba($color-dark, 0.8);
-  }
-
-  .kui-theme--dark & {
-    color: rgba($color-light, 0.8);
-  }
-
-  .kui-theme--dark &--faded {
-    color: rgba($color-light, 0.55);
-  }
-
-  .kui-theme--light &--faded {
-    color: rgba($color-dark, 0.55);
-  }
-
-  .kui-theme--dark &--disabled {
-    color: rgba($color-light, 0.25);
-  }
-
-  .kui-theme--light &--disabled {
-    color: rgba($color-dark, 0.25);
-  }
-
-  .kui-theme--dark & b {
-    color: $color-primary;
+  &--faded {
+    opacity: 0.65;
```

Comment:
is the color setting for the `--faded` condition replaced by the opacity in this case?

Reply:
yeah I've changed it to inherit the default text color, which has 0.85 opacity. So converting 0.55 / 0.85 = 0.65